docs/source/user/setup.rst

```diff
@@ -190,8 +191,8 @@ A popup will appear where you can define your "centre-id" and the type of datase
    The centre-id has to be lowercase and use alphanumeric characters only.
    The dropdown list shows all currently registered centre-ids on WIS2 as well as any centre-id you have already created in wis2box.
 
-There are 2 pre-defined dataset types for "weather/surface-based-observations/synop" and "weather/surface-based-observations/temp". 
-We recommend using these pre-defined dataset types to publish your "synop" and "temp" data, respectively. 
+There are 2 pre-defined dataset types for "weather/surface-based-observations/synop" and "weather/surface-based-observations/temp".
+We recommend using these pre-defined dataset types to publish your "synop" and "temp" data, respectively.
```

Comment:
There are more than 2 pre-defined dataset types in the wis2box-webapp nowadays. Can you please mention the dataset for the CAP alerts in this section